docs/development/extensions-core/google.md

```diff
@@ -23,7 +23,7 @@ title: "Google Cloud Storage"
   -->
 
 
-To use this Apache Druid extension, make sure to [include](../../development/extensions.md#loading-extensions) `druid-google-extensions` extension.
+To use this Apache Druid extension, make sure to [include](../../development/extensions.md#loading-extensions) `druid-google-extensions` extension and run druid processes with `GOOGLE_APPLICATION_CREDENTIALS=/path/to/service_account_keyfile` in the environment.
```

Comment:
Question to reviewers: Do all druid processes need this or just the ones that are talking to GCS ??

Reply:
I think you can set it for ones only who talk to GCS which are middleManagers, indexers, and historicals. But I guess setting this in all nodes doesn't harm as well.